### DISCUSSION TITLE
/MAT/LAW69/  run time very high

### DISCUSSION THREAD

**Discussion by aru2244** [OpenRadioss/OpenRadioss#947] [Users]
Hi
I was just trying a analysis with hyperelastic ogden model.  Using below stress starin curve it is taking a lot of run time 600hrs.
But when it tried with a curve given in your examples it is taking only 60hrs. What is the reason for that.How to reduce runtime 
in explicit
![image](https://user-images.githubusercontent.com/118266805/210055035-c0f5af3a-6cbc-4e7c-85e6-0d04065e5f21.png)

**Comment by OlivierWienholtz** [comment]
Hello,

You can check the time & timestep in Radioss output.
It is possible that your time step dropped  & you are doing a huge amount of explicit cycles.

**Reply by aru2244** [reply]
Thank You
